### PR TITLE
Upgraded Card from PF3 to PF4

### PR DESCRIPTION
**Pull request by DivyanshiSingh**
Fixes: #101 
1) Card feature is upgraded to pf4 in all files.
@jennydaman please review and let me know the required changes.

### CODE REVIEW

**Review by jennydaman — CHANGES_REQUESTED** [review]
Layout and positioning of options vdots are off.

image

For the future, it's better not to change code style e.g. single-quotes to double quotes — it is a huge merge conflict risk. There is an open issue for this: #168

File: src/components/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -111,49 +111,52 @@ class Dashboard extends Component {
...
-              >
+              <Button variant="primary" toRoute="/create">
                 Add Plugin
               </Button>
             </div>
           </div>
         </div>
         <div className="cards-pf dashboard-body">
-          <CardGrid>
+          <Grid className="container">
```
[the remark this answers is not in the document]
Comment (DivyanshiSingh) [reply src/components/Dashboard/Dashboard.js:137]: Ok, @jennydaman I will make the required changes.

[the remark this answers is not in the document]
Comment (DivyanshiSingh) [reply src/components/Dashboard/Dashboard.js]: done please review

**Review by jennydaman — APPROVED** [review]
Sorry for the wait @DivyanshiSingh 
Everything is looking good to me. Keep in mind for the future that you should try suppressing your IDE from reformatting the code style of entire files. It can lead to merge conflicts.
(Unless, that is, whatever future task you're working on is #168)

**Comment by DivyanshiSingh** [comment]
Ok sure, I will keep that in check.

**Pull request merged**